railties/lib/rails/generators/rails/app/templates/config/environments/test.rb.tt

```diff
@@ -8,7 +8,7 @@ require "active_support/core_ext/integer/time"
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
 
-  <%-# Spring executes the reloaders when files change. %>
```

Comment:
I thought this meant the comment wouldn't end up in the final evaluated template? Or does the `-` make it end up there?

Reply:
I think you're right, @kaspth ! Closing.